exercises/practice/state-of-tic-tac-toe/state_of_tic_tac_toe_test.go

```diff
@@ -0,0 +1,24 @@
+package stateoftictactoe
+
+import (
+	"testing"
+)
+
+func TestStateOfTicTacToe(t *testing.T) {
+	for _, c := range testCases {
+		result, err := StateOfTicTacToe(c.board)
+		if c.expectedErr != "" {
+			if err == nil || c.expectedErr != err.Error() {
+				t.Fatalf(`FAIL: %s
```

Comment:
When running the tests as subtests and especially with the `-v` flag, Go will print explicitly if each test passed or failed, so there's no need to include `FAIL:` or `PASS:` in the errors that the test itself returns.